.travis.yml

```diff
@@ -3,9 +3,11 @@ language: go
 go:
   - 1.5.2
 
-sudo: false
+sudo: required
 
 before_install:
+   - sudo apt-get update -y
+   - sudo apt-get install -y python-cffi
```

Comment:
arf. when `sudo: required` is enabled, we get a quite old ubuntu release (`precise` to be precise).
so, well, one will need to use `pip` instead.